culture/code-of-conduct.md

```diff
@@ -0,0 +1,124 @@
+
+The 2i2c Code of Conduct does not apply to users of any hubs that we manage on behalf of other organizations, though we encourage leaders in those communities to adopt a Code of Conduct for their hub infrastructure. The Code of Conduct **does** apply to any 2i2c Team Member that is acting in a community's hub.
+
+When in doubt, please {ref}`report <coc:reporting>` unacceptable behavior to us. If someone’s behavior outside of a 2i2c space makes you feel unsafe at 2i2c, that is absolutely relevant and actionable for us.
+
+
+## Enforcement
+
+We’ve categorized unacceptable behavior into abuse and unwelcoming behavior in the section above.
+
+If we witness or receive a report about abusive behavior, we will contact the perpetrator to have a conversation with them and verify what has transpired, and they will be removed from the 2i2c community. Their Slack account will be deactivated, and permissions will be removed from any 2i2c-related repositories. They will not be welcome in any physical or digital spaces covered by the 2i2c Code of Conduct.
+
+[This is the protocol](response-protocol.md) that 2i2c Directors and Steering Council members will use to respond to reports of code of conduct violations.
+
+(coc:reporting)=
+## Reporting
+
+If you see a violation of our code of conduct, please [report it to the 2i2c Executive Director](https://docs.google.com/forms/d/e/1FAIpQLSeKFuoghGzftqqgaa0xllZyocdUJsWZmoSaFFlerimg4n_Y8A/viewform?usp=sf_link).
+
+
+### Why should I report?
+
+* **You are responsible for making 2i2c a safe and comfortable space for everyone.** Everyone in our community shares this responsibility. 2i2c Team and Steering Council members are not around the online spaces or at 2i2c events all the time, so we cannot enforce the code of conduct without your help.
+* **The consequences for the 2i2c community of not reporting bad behavior outweigh the consequences for one person of reporting it.** We sometimes hear “I don’t want X person to meet consequences because I told someone about their bad behavior.” Consider the impact on everyone else at 2i2c of letting their behavior continue unchecked.
+* 2i2c only works as an open, participatory, community-driven community because of shared trust between community members. **Reporting code of conduct violations helps us identify when this trust is broken, to prevent that from happening in the future.**
+
+
+### Where and how to report
+
+Please report all code of conduct violations using [our reporting form](https://docs.google.com/forms/d/e/1FAIpQLSeKFuoghGzftqqgaa0xllZyocdUJsWZmoSaFFlerimg4n_Y8A/viewform?usp=sf_link). This is a short Google Form that will be sent to the Executive Director.
```

Comment:
If you are comfortable with it, @choldgraf: I think it would be appropriate to include your email in this statement too. A form can feel a bit bureaucratic, but also having an email there signals that there is a human on the other end responsible for reading it.

Reply:
good point - perhaps we should make a generic address like `director@2i2c.org`?